spec/graphql/object_type_spec.rb

```diff
@@ -24,5 +24,12 @@
       assert_equal(GraphQL::TypeKinds::NON_NULL, field.type.kind)
       assert_equal(GraphQL::TypeKinds::SCALAR, field.type.of_type.kind)
     end
+
+    it 'exposes defined field property' do
```

Comment:
I added that test to ensure that `DefinitionConfig#field` is assigning the property. `field_spec` doesn't test that flow.
I was going back & forth in my mind on whether to add this test, and then I had the thought that `DefinitionConfig#field` could either be moved into `Field` (possibly as a class method), or it can call a (not-yet-existing) method on `Field` to initialize the `Field` attributes. That way the `Field`-specific logic can be all in one place (same goes for tests).